apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.ts

```diff
@@ -11,7 +12,26 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     const institutions = await sqldb.queryRows(sql.select_institutions, InstitutionRowSchema);
-    res.send(AdministratorInstitutions({ institutions, resLocals: res.locals }));
+    const availableTimezones = await getAvailableTimezonesByName();
+    res.send(
+      AdministratorInstitutions({ institutions, availableTimezones, resLocals: res.locals }),
+    );
+  }),
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (req.body.__action === 'add_institution') {
```

Comment:
It's best to respond with a 400 if we see an unknown `__action`. Here's an example (note that we typically don't include the `locals: res.locals` bit in the "modern" form of this pattern):

https://github.com/PrairieLearn/PrairieLearn/blob/4f7b9a7fd6bec483511523157a67fe1980f76904/apps/prairielearn/src/pages/administratorCourses/administratorCourses.js#L125-L130

Reply:
Now throwing a 400 error in commit [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6). Was using `res.locals` at first, but removed this in commit [60f4565](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/60f456539fcea51223102ac47662dfa47bcc3dc0).